test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/ControllerSpec.scala

```diff
@@ -103,7 +107,7 @@ trait ControllerSpec extends WordSpec with Matchers with BeforeAndAfterAll with
               | microservice.upscan-initiate.upscan-store.expiry-time = 1
           """.stripMargin
           )
-        ) ++ additionalConfig
+        ).withFallback(additionalConfig)
```

Comment:
i think this is not a fallback, this is an additional config, so you should try to use the method that takes additional config

Reply:
Compiler has suggested to use this function through one warning message.